### PR TITLE
Create shared config, layers and translations

### LINKED ISSUE #358: Common configuration for WFP open data cube layers
We have a number of countries that will use a common set of layers that are accessed through WFP's open data cube instance as WMS layers. Currently, any change to a color ramp, legend text, etc. has to be made for each individual PRISM instance. 

Is there a way we can have these common configurations read from a single source so that we can ensure they are always in-sync and avoid errors in repeatedly updating them for each instance? If not a software solution, any suggestions on a file management solution to make this more efficient? Any thoughts @ericboucher?

### PR DESCRIPTION
**Pull request by ericboucher**
### Description

This fixes #358 by adding:
- default config
- shared layers
- shared legends
- shared translations

@wadhwamatic the configs are deeply merged, by default you have access to any layers in the shared layers and can just call them by `id` from your prism.json file as usual. If you need to tweak the name or the legends or else specifically for a country, you can do so by creating a layer with the same id with only the updated elements.

@wadhwamatic for now I simply created one `shared/layers.json` file but we can defiitely split it by products maybe for ease of use (no need to decide now).
 

## How to test the feature:

- [ ] make sure that translation works as expected
- [ ] test moving some layer definitions to the shared space.

## Checklist - did you ...

Test your changes with

- [ ] `REACT_APP_COUNTRY=rbd yarn start`
- [ ] `REACT_APP_COUNTRY=cambodia yarn start`
- [ ] `REACT_APP_COUNTRY=mozambique yarn start`
- [ ] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

**Comment by wadhwamatic** [comment]
Thanks @ericboucher, this will be a big help when ready. A few things:
1. The current layers.json in shared has errors. For example, the base_url and styles are incorrect
2. What if some elements in the layer config are common, but others are not? For example, the `levels` property in `chart_data` is unique per country, but most other properties are common. We also would likely use one set of layer styles in more arid regions, and another in wetter regions. Would we need to then config those layers uniquely per country or could we find some way to inherit some properties, but not all?
3. I'm thinking about how to deploy this. If prism.json refers to a  layer ID that exists in both the local layers.json file, and the shared one, which will be used?

**Comment by ericboucher** [comment]
@wadhwamatic:
The current behavior is that local files overrides the shared one (deep merge) so:
1. we can start from an empty file and you create the base layers that you want. As mentioned above, we could even split this into multiple files for clarity
2. you can simply configure that specific parameter in the local config and let the rest come from the shared config
3. both, with the local config taking precedent.

Here is an example:

shared file:
{url: hello, legend: a,b,c, map: {bbox: 1234}}

local file
{url: prism, map: {bbox: 4567}}

result:
{url: prism, legend: a,b,c, map: {bbox: 4567}}

File: frontend/src/config/mozambique/layers.json [hunk 1]
```diff
@@ -1805,14 +1805,6 @@
   },
   "rainfall_agg_1month": {
```
[the remark this answers is not in the document]
Comment (wadhwamatic) [reply frontend/src/config/mozambique/layers.json:1806]: @ericboucher - the title text is slightly different for Mozambique.

[the remark this answers is not in the document]
Comment (ericboucher) [reply frontend/src/config/mozambique/layers.json:1806]: ok, does it need to be?

[the remark this answers is not in the document]
Comment (wadhwamatic) [reply frontend/src/config/mozambique/layers.json:1806]: We won't always include CHIRPS in the title. It's only for countries where we have local rainfall data and CHIRPS rainfall data.

**Comment by wadhwamatic** [comment]
@ericboucher - before the shared layer concept, the order of layers in layers.json controlled the order of the 'select charts' dropdown list. Now, the shared layers are first, and then local layers. Can we reverse this? For Mozambique, I want the blended layers to show up first, then CHIRPS layers (which is how it worked before this change).

**Comment by wadhwamatic** [comment]
@ericboucher - ok, I finished setting this up for Mozambique and I think it should be straightforward to apply to others. Making those configuration changes for all will take a while so I'd rather just do Mozambique in this one.

Two things:
1. We need to update documentation to capture this change
2. There's no longer a way to control layer order in charts. See my comment https://github.com/WFP-VAM/prism-app/pull/1157#issuecomment-2052635716

**Comment by ericboucher** [comment]
@wadhwamatic I updated the readme, feel free to adjust as needed and merge when you think it's ready

**Review by echaidemenos — APPROVED** [review]
Everything looks and works OK for me here. Maybe we should split "General" and mozambique specific translations and use both `mozambique/translations.json` and `shared/portuguese.json`. Or just remove `mozambique/translations.json` for now

**Comment by ericboucher** [comment]
Yep or move `mozambique/translations.json` to `mozambique/translations/portuguese.json` or similar

**Comment by wadhwamatic** [comment]
I deleted the local Mozambique translation file for now. If another Portuguese-speaking country comes along, we might need some level of localization for Moz vs a second country but for now will just keep a single shared Portuguese translation file.

**Pull request merged**